.github/workflows/tag-checks.yml

```diff
@@ -16,7 +16,7 @@ jobs:
       matrix:
         # On version change, add new version here
         # On breaking changes, by definition past version mays no longer work. Remove previous versions from the github action.
-        tag: []
+        tag: [v0.3.2]
```

Comment:
What exactly will this do ?

Reply:
It will test this version. v0.3.1 had a broken test so it didn't make sense to retest it.